en/hardware/porting_guide_config.md

```diff
@@ -0,0 +1,46 @@
+# PX4 Board Configuration (Kconfig)
```

Comment:
Note, I renamed this. This is board configuration. It might be used by a Porting Guide, but it isn't specific to porting. We might want to rename the file too if you are OK with that?

Reply:
Indeed makes sense it can also be used for reconfiguration as well.